package.json

```diff
@@ -51,7 +51,7 @@
     "@docusaurus/module-type-aliases": "^3.1.1",
     "@docusaurus/tsconfig": "3.1.1",
     "@prettier/sync": "^0.5.1",
-    "cypress": "^13.15.0",
+    "cypress": "^14.3.2",
```

Comment:
I'm not sure if this PR should be bumping the version of Cypress or if it would be better to separate this out.
Also Node.js should be bumped >= 20, since Node.js is now EOL.

Reply:
yeah good point, let's leave the dependencies alone here.